templates/automotive/index.html

```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
+      <p>You innovate. We maintain.</p>
+      <p>Extend ECU life and lifetime value with platform upgrades for new services and solutions.</p>
+      <p>Count on Canonical security coverage. Regular updates. Automated testing. Continuous deployment. Progressive releases.</p>
+      <p>A new world of constant car improvement.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/1d5f293c-compliance.svg",
+          alt="",
+          height="200",
+          width="330",
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--five">Open source licensing</h2>
+      <p class="p-heading--two">Straightforward compliance</p>
+      <p>Confidently integrate open source components into your proprietary solutions.</p>
+      <p>Source identification for all software components, all the time.</p>
+      <p>Simplify compliance. Streamline certification. Protect innovation.</p>
+    </div>
+    <div class="col-6">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/338124d1-shields-security.svg",
+          alt="",
+          height="200",
+          width="350",
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--five">Connected car security</h2>
+      <p class="p-heading--two">Hardened apps. Hardened OS.</p>
+      <p>Your customers want apps. Your business needs security. Get both on Ubuntu.</p>
+      <p>Secure boot, <a href="/engage/iot-disk-encryption">full disk encryption</a> and <a href="https://snapcraft.io/docs/assertions" class="p-link--external">signed software packages</a> guarantee system integrity.</p>
+      <p><a href="https://snapcraft.io/docs/snap-confinement" class="p-link--external">Snap confinement</a> limits blast radius, keeps your apps apart, and controls access to hardware.</p>
+      <p>Curate your app store from global suppliers, with constant app security scanning and updates.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="p-heading--five">Software strategy acceleration</h2>
+      <p class="p-heading--two">Trusted by executives. Loved by gearheads.</p>
+      <p>The <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf" class="p-link--external">best developers choose Ubuntu</a>.</p>
+      <p>The biggest pool of talent. The broadest range of tools.  The community that counts.</p>
+      <p>One company stands behind it all &mdash; Canonical.</p>
+    </div>
+    <div class="col-6">
+      <h2 class="p-heading--five">Domain experts</h2>
+      <p class="p-heading--two">Innovate in automotive with Canonical</p>
+      <p>The experts in open source operations, from cloud to desktop to device. The publishers of Ubuntu.</p>
+      <p>Get to market faster.</p>
```

Comment:
these three paragraphs are ticked bullets in the design

Reply:
Good spot, done